moped-database/migrations/1649874244000_alter_project_list_view/down.sql

```diff
@@ -0,0 +1,41 @@
+drop view project_list_view;
+
+CREATE VIEW public.project_list_view AS
```

Comment:
This is the `down.sql` which is putting the old view back in place, so I totally am on board with just taking the definition as spit out of a DB client, `graphql-engine` or `pg_dump` most ideally, but if it's written by hand, the same sort of suggestions as the `up.sql` apply.